src/test/groovy/karuna/karuna_backend/Constants.groovy

```diff
@@ -0,0 +1,23 @@
+package karuna.karuna_backend
+
+trait Constants {
+
+    static final String PAGE = "HOME"
+    static final String KEY = "home.key"
+    static final String VALUE = "Home value"
+
+    static final String TO = "test@test.pl"
+    static final String SUBJECT = "Test KARUNA"
+    static final String CONTENT_TEXT = "To tylko test"
```

Comment:
should be EN?

Reply:
why? its only test data, variables should be english

Reply:
ok